src/main.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use(value_t)]
+extern crate clap;
```

Comment:
I don't believe you actually need this, you should be able to just write `use clap::value_t` these days; this is an old-style macro import.

Reply:
Oh yes, that worked, thank you!